### ISSUE TITLE
populating the database as root, gives access denied error

### ISSUE THREAD

**Issue by ayush0x00** [LibreHealthIO/lh-ehr-laravel#15]
**Describe the bug**
As mentioned in the readme, when the .env file is configured with username as root, it gives`SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using pas sword: NO) (SQL: select * from information_schema.tables where table_schema = test and table_name = migrations)` error.

**To Reproduce**
Follow the installation process. The error appears as soon as we populate the database with seed.

**Expected behavior**
The database should populate without any errors

**Desktop (please complete the following information):**
 - OS: Linux Mint
 - Browser Brave

**Comment by ayush0x00** [comment]
The fix PR is failing checks even though only readme was updated. Is it fine? @muarachmann

**Comment by muarachmann** [comment]
> The fix PR is failing checks even though only readme was updated. Is it fine? @muarachmann

Yes I will fix the build in my PR #13. I made a few changes. Also I don't think this is an issue very much as it's a trivial stuff for everyone to modify their `.env` once they do a `cp .env.example .env` which I have made clear in the readme. That said, this should rather be done in something like the **troubleshoot.md** so that it helps others that get stuck.
